### PR TITLE
Resolve Static Code Analysis Issues

### LINKED ISSUE #31: Multiple Warnings From Qodana
**Describe the bug**
After adding Qodana to the CI, it exposed a few dozen issues with the codebase (some of which we've been aware of) that should be resolved.

**To Reproduce**
https://github.com/RIT-Launch-Initiative/GSW/runs/40532921716

**Expected behavior**
No complaints from Qodana

### PR DESCRIPTION
**Pull request by AarC10**
This closes #31

# Description
Add error handling, fix a few of the logging statements (more should be fixed). We should probably not have an abstraction over the logging library, since I did have some issues debugging at some point but thats a whole other issue. Remove unused files

# How Has This Been Tested?
Describe how you tested this change and any special setup configurations in case someone else wants to reproduce this.  

# Checklist:
- [x] New functionality is documented in the necessary spots (i.e new functions documented in the header)
- [x] Unit tests cover any new functionality or edge cases that the PR was meant to resolve (if applicable) 
- [x] The CI checks are passing
- [x] I reviewed my own code in the GitHub diff and am sure that each change is intentional

### CODE REVIEW

**Review by Smartinez3324 — COMMENTED** [review]
LFG 👍

**Review by Smartinez3324 — COMMENTED** [review]
good

**Review by Smartinez3324 — APPROVED** [review]
looks amazing 🤩

**Comment by AarC10** [comment]
> looks amazing 🤩

Now if only your K/D was just as amazing 🥱

File: cmd/gsw_service.go [hunk 1]
```diff
@@ -145,19 +147,22 @@ func main() {
...
-		fmt.Printf("Received signal: %s\n", sig)
+		logger.Debug("Received signal: ", zap.String("signal", sig.String()))
 		cancel()
 	}()
 
 	configWriter, err := vcmInitialize(config)
 	if err != nil {
-		logger.Info("Exiting GSW...")
+		logger.Panic("Exiting GSW...")
 		return
```
Comment (apenza) [inline cmd/gsw_service.go:157]: return after panic?

**Review by apenza — APPROVED** [review]

File: cmd/gsw_service.go (hunk 1 above)
Comment (AarC10) [reply cmd/gsw_service.go:157]: I assumed panic just logs at the panic level? But regardless, we would be exiting main anyways so GSW would exit early
